modules/ivc_champva/spec/models/vha_10_7959a_spec.rb

```diff
@@ -15,6 +15,7 @@
       'veteran' => {
         'full_name' => { 'first' => 'John', 'middle' => 'P', 'last' => 'Doe' },
         'va_claim_number' => '123456789',
+        'ssn_or_tin' => '123456789',
         'address' => { 'country' => 'USA', 'postal_code' => '12345' }
       },
```

Comment:
There is no `veteran` block in the data coming from 10-7959a. We'll want this section to be:

```javascript
{
  'applicant_name' => { 'first' => 'John', 'middle' => 'P', 'last' => 'Doe' },
  'applicant_member_number' => '123456789',
  'applicant_address' => { 'country' => 'USA', 'postal_code' => '12345' },
}
```

Reply:
I thought you were going to change it on your commit